src/settings/src/persistentinfo.cpp

```diff
@@ -186,7 +186,8 @@ void PersistentInfo::UpdateSettings()
             const auto id = generateIdFromUuid();
             sessionSettings_->beginGroup( "OpenFiles" );
             std::vector<std::tuple<QString, uint64_t, QString>> openFiles;
-            int size = sessionSettings_->beginReadArray( "openFiles" );
+            const int size = sessionSettings_->beginReadArray( "openFiles" );
```

Comment:
This place is reading settings, so not performance critical. Can stay without reserve

Reply:
fix it https://github.com/variar/klogg/commit/a4f9cad9eceff87a667f1c6cec939f383153a6e3